diesel/src/migrations/mod.rs

```diff
@@ -135,7 +137,8 @@ fn migration_with_version(ver: &str) -> Result<Box<Migration>, MigrationError> {
     }
 }
 
-fn create_schema_migrations_table_if_needed(conn: &Connection) -> QueryResult<usize> {
+#[doc(hidden)]
+pub fn create_schema_migrations_table_if_needed(conn: &Connection) -> QueryResult<usize> {
```

Comment:
Wait this is in migrations.rs. Did I not finish my metadata work? Shit maybe I shouldn't have deleted all my branches